doc/Caliptra.md

```diff
@@ -87,6 +87,14 @@ In this version of the specification, the desired capabilities address the basic
 
 Caliptra implements the DICE Protection Environment (DPE) API, allowing it to derive and wield a DICE identity on behalf of other elements within the SoC. Use cases for this API include serving as a signing oracle for a Security Protocol and Data Model (SPDM) responder that is executing in the SoC application processor, as well as authentication to a discrete TPM device.
 
+### SVN-bound key derivation via hash chains
```

Comment:
I'm thinking we should hold off on this PR until we have published the 1.0 spec. We should also decide how we want to handle spec changes post-1.0. Should the main branch here just accumulate any spec changes for future releases? Or should main always be for the most recent release.

Reply:
Closing this in favor of https://github.com/chipsalliance/Caliptra/pull/171